resources/assets/themes/base.scss

```diff
@@ -39,18 +46,29 @@ $form-label-font-weight: $font-weight-bold;
 @import "~bootstrap/scss/nav";
 @import "~bootstrap/scss/navbar";
 @import "~bootstrap/scss/card";
+// @import "~bootstrap/scss/accordion";
```

Comment:
Why have you added the commeted out imports in the next couple of lines? I'd suggest to either leave another comment to explain them or to remove them again.

Reply:
added comments

Reply:
I would mandate for removing them, if unused.